test/cli.js

```diff
@@ -364,6 +364,14 @@ test('bails when config contains `"tap": true` and `"watch": true`', t => {
 	});
 });
 
+test('bails when config does not contain `--concurrency` and `-c` values', t => {
```

Comment:
@novemberborn This works! Can you tell me what all cases to cover in `test/fixture/concurrency/test.js` ?